### PR TITLE
docs: Update changelog for v2.3.0

### PR DESCRIPTION
**Pull request by corycaywood**
# Description
Updates to the change log for 2.3.0

# Checklist
- [x] I have read the [contributing guidelines](../CONTRIBUTING.md).
- [ ] I wrote/updated tests for new/changed code
- [x] I removed all sensitive data before every commit, including API endpoints and keys
- [x] I ran `fastlane ci` without errors

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,18 +1,21 @@
 ## CHANGELOG
 
-### 2.x.x (In progress)
+### 2.3.0 (202010-22)
```
Comment (ts-leojoseph-r) [inline CHANGELOG.md:3]: Looks like that date format is bit different

**Review by ts-leojoseph-r — APPROVED** [review]
Looks good to me :+1:

**Review by ts-leojoseph-r — COMMENTED** [review]
We need to update this value as well in the plist ?

```
<key>RMAHostAppUserAgentInfo</key>
	<string>MiniApp Demo App/2.2.0</string>
```

File: CHANGELOG.md (hunk 1 above)
Comment (corycaywood) [reply CHANGELOG.md:3]: Oh, thanks for spotting that. I made a typo 😓

**Review by Climbatize — APPROVED** [review]
LGTM

**Pull request merged**